nomad-jobs/terraform/main.tf

```diff
@@ -1,58 +1,85 @@
+  job_name    = "fabio-manage"
+  run         = "${var.fabio_compute["run"]}"
+  node_class  = "${var.fabio_compute["node_class"]}"
+  region      = "${var.region}"
+  datacenters = "${var.datacenters}"
 }
 
-variable "prometheus_cpu_limit" {
-  description = "CPU resource limit"
-  default = "450"
+module "fabio-default" {
+  source      = "github.com/ketzacoatl/terraform-cluster-common-nomad-jobs//fabio?ref=0.1"
+  job_name    = "fabio-default"
+  run         = "${var.fabio_default["run"]}"
+  node_class  = "${var.fabio_default["node_class"]}"
+  region      = "${var.region}"
+  datacenters = "${var.datacenters}"
 }
 
-variable "prometheus_mem_limit" {
-  description = "Memory resource limit"
-  default = "512"
+module "grafana" {
+  source      = "github.com/ketzacoatl/terraform-cluster-common-nomad-jobs//grafana?ref=0.1"
+  run         = "${var.grafana["run"]}"
+  region      = "${var.region}"
+  datacenters = "${var.datacenters}"
+  node_class  = "${var.grafana["node_class"]}"
 }
 
-variable "prometheus_net_limit" {
-  description = "Network resource limit"
-  default = "5"
+module "hashi-ui" {
+  source      = "github.com/ketzacoatl/terraform-cluster-common-nomad-jobs//hashi-ui?ref=0.1"
+  run         = "${var.hashi-ui["run"]}"
+  region      = "${var.region}"
+  datacenters = "${var.datacenters}"
 }
 
-# create a list of template_file data sources with init for each instance
-data "template_file" "prometheus" {
-  count    = "${var.run_prometheus}"
-  template = "${file("../job-templates/prometheus-exec.tpl")}"
-  vars {
-    region      = "${var.region}"
-    datacenters = ["${var.datacenters}"]
-    version     = "${var.prometheus_version}"
-    checksum    = "${var.prometheus_checksum}"
-    cpu_limit   = "${var.prometheus_cpu_limit}"
-    mem_limit   = "${var.prometheus_mem_limit}"
-    net_limit   = "${var.prometheus_net_limit}"
-  }
+module "ladder" {
+  source      = "github.com/ketzacoatl/terraform-cluster-common-nomad-jobs//ladder-docker?ref=0.1"
+  run         = "${var.ladder["run"]}"
+  region      = "${var.region}"
+  datacenters = "${var.datacenters}"
 }
 
-provider "nomad" {
-  address = "nomad-server.service.consul"
-  region  = "${var.region}"
+module "node_exporter" {
+  source      = "github.com/ketzacoatl/terraform-cluster-common-nomad-jobs//node_exporter?ref=0.1"
+  run         = "${var.node_exporter["run"]}"
+  region      = "${var.region}"
+  datacenters = "${var.datacenters}"
+  job_name    = "${var.node_exporter["job_name"]}"
+}
+
+module "nomad-metrics" {
+  source      = "github.com/ketzacoatl/terraform-cluster-common-nomad-jobs//nomad-metrics?ref=0.1"
+  run         = "${var.nomad-metrics["run"]}"
+  region      = "${var.region}"
+  datacenters = "${var.datacenters}"
+  node_class  = "${var.nomad-metrics["node_class"]}"
 }
 
-resource "nomad_job" "prometheus" {
-  jobspec = "${data.template_file.prometheus.rendered}"
+module "sysbench" {
```

Comment:
while I want this module available for use when we would use it.. we probably don't want to include this in the default / example env.